### PR TITLE
Reinstated antsIntroduction script

### PR DESCRIPTION
**Pull request by TheChymera**
## List of changes proposed in this PR (pull-request)
As discussed in https://github.com/nipy/nipype/issues/3197#issuecomment-626318459

## Acknowledgment

- [x] \(Mandatory\) I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by oesteban** [comment]
`make specs` ?

**Comment by TheChymera** [comment]
@oesteban hm, I can't seem to run that:

```
chymera@silenthost ~/src/nipype $ make specs
Checking specs and autogenerating spec tests
env PYTHONPATH=".:" python tools/checkspecs.py
Traceback (most recent call last):
  File "tools/checkspecs.py", line 11, in <module>
    import black
ModuleNotFoundError: No module named 'black'
make: *** [Makefile:73: specs] Error 1
```

**Comment by effigies** [comment]
`black` is in the `specs` and `dev` extra.

**Comment by TheChymera** [comment]
@effigies it still fails as it tries to import the deprecated `sphinxcontrib-napoleon`. As per [the very first section on the upstream page](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/) this should now be imported as `sphinx.ext.napoleon`.

I'd fix this myself, but I'm unsure whether that wouldn't break the nipype docs. It seems there's some napoleon hacking in nipype: https://github.com/nipy/nipype/blob/d2a48e1ebcb1a876bb48a63164dfc46323c45329/doc/conf.py#L125

**Comment by TheChymera** [comment]
@oesteban any ideas?

**Comment by satra** [comment]
@TheChymera - according to this page (https://github.com/ANTsX/ANTs/wiki/Deprecated-or-legacy-ANTs-programs) antsIntroduction.sh is indeed considered deprecated or legacy and other scripts are recommended. any particular reason why we should support this?

**Comment by TheChymera** [comment]
@satra recent ANTs still ships this, and ofc I assume many people won't be using the very latest ANTs even when they drop it. Some of my colleagues still use the function, and I use it in an article in which I benchmark it against a newer workflow.

File: nipype/interfaces/ants/registration.py [hunk 1]
```diff
@@ -1874,3 +1874,131 @@ def _list_outputs(self):
...
+        exists=True, desc='inverse warp field (prefix_InverseWarp.nii)')
+    input_file = File(exists=True, desc='input image (prefix_repaired.nii)')
+    output_file = File(exists=True, desc='output image (prefix_deformed.nii)')
+
+
+class antsIntroduction(ANTSCommand):
+    """Uses ANTS to generate matrices to warp data from one space to another.
+    Examples
+    --------
+    >>> from nipype.interfaces.ants.legacy import antsIntroduction
```
Comment (effigies) [inline nipype/interfaces/ants/registration.py:1971]:
```suggestion
    >>> from nipype.interfaces.ants.registration import antsIntroduction
```

**Comment by satra** [comment]
@effigies @TheChymera - since ants itself considers this legacy, should we reinstate the legacy package and keep it there.

**Comment by effigies** [comment]
Okay. I'll open a PR.

**Comment by TheChymera** [comment]
@effigies @satra thank you.

**Pull request closed without merging**